### ISSUE TITLE
Undefined offset notice

### ISSUE THREAD

**Issue by ghost** [auraphp/Aura.Accept#8]
Hi
I have tested [this](https://github.com/auraphp/Aura.Accept#negotiating-media-types) example in README.md file. the result is ok but there is a notice.
![screenshot from 2016-12-15 19-40-02](https://cloud.githubusercontent.com/assets/15300551/21231644/3d9dd42e-c2fe-11e6-9ff8-d945aaaf5c19.png)

**Comment by ghost** [comment]
@harikt I think this should be fix and I am sure it is not so hard to fixing this, is it?  ;)

**Comment by harikt** [comment]
@sharifzadesina  I wonder whether that was a typo in the docs ( have not put the additional star ) . So `application/xml;q=1.0,text/csv;q=0.5,*;q=0.1` will become `application/xml;q=1.0,text/csv;q=0.5,*/*;q=0.1` . I have a fix if the doc is correct. But need more research before I merge the same.

Below is the fix for the same 

```php
class MediaValue ....

    protected function init()
    {
        // list($this->type, $this->subtype) = explode('/', $this->value);
        $values = explode('/', $this->value);
        $this->type = $values[0];
        $this->subtype = isset($values[1]) ? $values[1] : '*';
    }
```

@pmjones any different thought?

**Comment by ghost** [comment]
@harikt You are right according to the [RFC7231](https://tools.ietf.org/html/rfc7231#section-5.3.2) this should be `*/*`. but unfortunately chrome sends `*`.
this is not cause of doc dude, this is cause of chrome which doesn't follow standard syntax. I am not sure but it seems this bug was fixed in new version of chrome.

**Comment by harikt** [comment]
Thank you @sharifzadesina for linking to the RFC. 

For reference to others : fixed via https://github.com/auraphp/Aura.Accept/pull/9 . In case of better fixes send a different one.

Thank you.

**Comment by pmjones** [comment]
Looks good @harikt !  Thanks, @sharifzadesina , for the report.

Fixed in #9
